src/store/modules/mobile.js

```diff
@@ -23,7 +23,9 @@ const derivePasswordKey = async (password, salt) => {
 
 export default {
   state: {
-    keystore: false,
+    masterKey: null,
+    hasMasterKey: false,
+    keystore: null,
```

Comment:
`hasMasterKey` is derived from `masterKey`? `masterKey` is the same as `keystore` but in Cordova? Don't make state bigger without necessity.